soc/arm/st_stm32/stm32f3/Kconfig.defconfig.stm32f303x8

```diff
@@ -1,9 +1,9 @@
 # ST Microelectronics STM32F303x8 MCU
 
-#  Copyright (c) 2020 Sebastian Schwabe 
-#							<sebastian.schwabe@mailbox.tu-dresden.de>,
-# 					  Fabian Paschke
-#					   		<fabian.paschke@eas.iis.fraunhofer.de>
+#  Copyright (c) 2020 Sebastian Schwabe
+#  <sebastian.schwabe@mailbox.tu-dresden.de>
+#  Fabian Paschke
+#  <fabian.paschke@eas.iis.fraunhofer.de>
```

Comment:
Please fix this in the initial commit

Reply:
How can I change an outdated commit? I removed the initial commit and add the three new one for dts, soc and board.

Reply:
https://fle.github.io/git-tip-keep-your-branch-clean-with-fixup-and-autosquash.html

TL;DR:
-add a new commit ths fixes the issue. As commit title, use: `fixup! <sha of the commit to be fixed>`
-Repeat for each fix to be done on the branch
-Once you're done: `git rebase -i --autosquash HEAD~<numbers of commits of your branch>`